packages/framer-motion/src/animation/animators/AcceleratedAnimation.ts

```diff
@@ -159,6 +168,22 @@ export class AcceleratedAnimation<
             return false
         }
 
+        /**
+         * If the user has provided an easing function name that isn't supported
+         * by WAAPI (like "anticipate"), we need to provide the corressponding
+         * function. This will later get converted to a linear() easing function.
+         */
+        if (
+            typeof ease === "string" &&
+            supportsLinearEasing() &&
+            ease in unsupportedEasingFunctions
+        ) {
+            ease =
+                unsupportedEasingFunctions[
+                    ease as keyof typeof unsupportedEasingFunctions
+                ]
```

Comment:
If you made a type guard like:
```typescript
function isUnsupportedEase(key: string): key is keyof unsupportedEasintFunctions {
  return key in unsupportedEasintFunctions
}
```

You could avoid the casts.

Reply:
ty will implement